src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,28 @@
 package mate.jdbc;
 
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Driver;
+import mate.jdbc.service.DriverService;
+
 public class Main {
+
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        DriverService driverService = (DriverService) injector
+                .getInstance(DriverService.class);
+        driverService.getAll().forEach(System.out::println);
+        Driver ivan = new Driver();
+        ivan.setName("Ivan");
+        ivan.setLicenseNumber("22500335");
+        System.out.println(driverService.create(ivan));
+        driverService.getAll().forEach(System.out::println);
+        ivan.setLicenseNumber("22550335");
+        ivan.setId(ivan.getId());
+        System.out.println(driverService.update(ivan));
+        driverService.getAll().forEach(System.out::println);
+        System.out.println(driverService.get(3L));
```

Comment:
is this an example for a non-existing driver? please add some info in print methods